test/tests/functional/pbs_sched_load_balancing.py

```diff
@@ -0,0 +1,75 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2019 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class TestSchedLoadBalancing(TestFunctional):
+    """
+    Test suite for PBSPro's Scheduler load balancing
+    """
+    def setUp(self):
+        """
+        Set scheduler load_balancing
+        """
+        TestFunctional.setUp(self)
+        self.scheduler.set_sched_config({'load_balancing': 'true	ALL'})
```

Comment:
since you have just one test right now, it's probably best not to turn on load balancing in setUp(), if we add more tests to this suite and all of them need it to be on by default then we can add this to setUp().

Reply:
But this is a Scheduler's load_balancing test suite and that set up is to turn on the feature. So later also if we add any test cases under this suite they probably need this.